packages/renderer/src/lib/ui/NavSection.svelte

```diff
@@ -26,6 +33,13 @@ function fadeSlide(node: any, { delay = 0, duration = 400, easing = cubicOut })
       `padding-bottom: ${t * paddingBottom}px;`,
   };
 }
+
+onMount(() => {
+  // collapse by default if > 3 items
+  if ($count > 3) {
```

Comment:
maybe a property ( defaulted to 3) but we could have different values for kube or extensions

Reply:
sure